pyproject.toml

```diff
@@ -40,14 +40,19 @@ caikit-render-interfaces = "caikit.runtime.dump_services:main"
 
 ## Runtime Extra Sets ##
 
+runtime-base = [
```

Comment:
Hmm, the only reason why I didn't do that is because `prometheus_client` is a base requirement for the runtime and not the core? Not sure if that makes a difference

Reply:
🤞 that we can remove this req with the move to open telemetry... at some point.

But what do we do about requirements that in general are required for metrics and would be required for either the http or grpc server? `runtime-metrics`?